R/README.md

```diff
@@ -1,6 +1,16 @@
 # R on Spark
 
 SparkR is an R package that provides a light-weight frontend to use Spark from R.
+### Installing sparkR
+
+Libraries of sparkR need to be created in `$SPARK_HOME/R/lib`. This can be done by running the script `$SPARK_HOME/R/install-dev.sh`.
```

Comment:
If one were running from R studio with the steps in `Using SparkR from RStudio` below he wouldn't have to install or run install-dev.sh though - could we clarify that?